src/submission/signature/index.jsx

```diff
@@ -16,10 +16,7 @@ const showWarning = props => {
 }
 
 const Signature = props => {
-  let isDisabled =
-    props.status.code >= VALIDATED && props.status.code !== SIGNED
-      ? false
-      : true
+  let isDisabled = props.status.code === VALIDATED ? false : true
```

Comment:
The previous `&&` check seemed redundant. `VALIDATED` is immediately followed by `SIGNED`.

Reply:
I believe it was a safeguard against a user reaching the submission page when the file wasn't ready to be submitted (thus the `>` rather than equality check). The router should prevent this, but this was a failsafe.